plugin/pkg/admission/namespace/lifecycle/admission.go

```diff
@@ -53,7 +53,7 @@ func (l *lifecycle) Admit(a admission.Attributes) (err error) {
 
 	// prevent deletion of immortal namespaces
 	if a.GetOperation() == admission.Delete && a.GetKind() == "Namespace" && l.immortalNamespaces.Has(a.GetName()) {
-		return errors.NewForbidden(a.GetKind(), a.GetName(), fmt.Errorf("namespace can never be deleted"))
+		return errors.NewForbidden(a.GetKind(), a.GetName(), fmt.Errorf("an immortal namespace can never be deleted"))
```

Comment:
I'm not sure we want to introduce a concept of "immortal namespaces" because people will want to make them.  Maybe this should just say "this namespace may not be deleted".  Key words "this" and "may not be" as opposed to "can never be".

@derekwaynecarr